apps/website/src/app/[locale]/page.tsx

```
@@ -10,21 +10,25 @@ import {LanguageSelector} from '@/components/LanguageSelector';
 import {JoinSectionImmersive} from '@/components/JoinSectionImmersive';
 import {AboutSectionBento} from '@/components/AboutSectionBento';
 import {BrandKeywordHighlight} from '@/components/BrandKeywordHighlight';
+import {EventsSection} from '@/components/EventsSection';
 import {motion} from 'framer-motion';
+import {siteConfig} from '@/config/site';
 
 export default function IndexPage({params}: PageProps<'/[locale]'>) {
   const {} = use(params);
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)